bench/bench.c

```diff
@@ -24,9 +24,40 @@
 
 #include <assert.h>
 #include <stdio.h>
-#include <sys/time.h>
 
 #include "bench.h"
+#include "../src/utils.h"
+
+#ifndef _MSC_VER
+#include <sys/time.h>
+#else
+#define WIN32_LEAN_AND_MEAN
+#include <windows.h>
+#include <stdint.h>
+
+struct timeval {
+    long tv_sec, tv_usec;
+};
+
+static int
+gettimeofday(struct timeval *tv, void *unused)
```

Comment:
Hmm, maybe we should just use `clock()`, which is not great but is standard C, instead?

This is fine, anyway.